src/compiler/transformers/transform-utils.ts

```diff
@@ -727,18 +733,23 @@ export const createRequireStatement = (importFnNames: string[], importPath: stri
         importAs = splt[1];
         importFnName = splt[0];
       }
-      return ts.createBindingElement(undefined, importFnName, importAs);
+      return ts.factory.createBindingElement(undefined, importFnName, importAs);
     })
   );
 
-  return ts.createVariableStatement(
+  return ts.factory.createVariableStatement(
     undefined,
-    ts.createVariableDeclarationList(
+    ts.factory.createVariableDeclarationList(
       [
-        ts.createVariableDeclaration(
+        ts.factory.createVariableDeclaration(
           importBinding,
           undefined,
-          ts.factory.createCallExpression(ts.factory.createIdentifier('require'), [], [ts.createLiteral(importPath)])
+          undefined,
```

Comment:
We add `undefined` here because the fn sig changes when we move to the `factory` version
```diff
- createClassExpression: (modifiers: readonly Modifier[] | undefined, name: string | Identifier | undefined, typeParameters: readonly TypeParameterDeclaration[] | undefined, heritageClauses: readonly HeritageClause[] | undefined, members: readonly ClassElement[]) => ClassExpression;
+ createClassExpression: (decorators: readonly Decorator[] | undefined, modifiers: readonly Modifier[] | undefined, name: string | Identifier | undefined, typeParameters: readonly TypeParameterDeclaration[] | undefined, heritageClauses: readonly HeritageClause[] | undefined, members: readonly ClassElement[]): ClassExpression;
```